source/administration/upgrade.rst

```diff
@@ -6,9 +6,9 @@ In most cases you can upgrade Mattermost Server in a few minutes, but the upgrad
 Which Upgrade Instructions to Use
 ---------------------------------
 
-If you are upgrading from version 3.7.0 or greater
+If you are upgrading from version 3.8.0 or greater
```

Comment:
The way it's currently set up, supported versions are in this doc and unsupported versions are in the legacy doc.